scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -182,6 +182,7 @@ task CollectCounts {
     File ref_fasta
     File ref_fasta_fai
     File ref_fasta_dict
+    Boolean? enable_indexing
```

Comment:
I think we decided on not specifying values for optional parameters in this way as a matter of style across all CNV WDLs, so that default values for parameters that are simply passed through to the command line would be located in the command-line invocation itself.  Maybe we can change everything consistently in the WDL 1.0 update PR if it makes sense to do so.

Reply:
Wait, also see https://github.com/broadinstitute/gatk/issues/2858#issuecomment-328652790.

Reply:
Interesting. I am unsure how the use of optional works in this case, ie what is the difference between `Boolean enable_indexing = false` and `Boolean? enable_indexing = false`. Both are allowed and we use the former in the SV pipeline, although I have not tested that it can be overridden in the json.